common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -132,7 +137,7 @@ public class RemoteBlockPushResolver implements MergedShuffleFileManager {
   final File recoveryFile;
 
   @VisibleForTesting
-  final DB db;
+  DB db;
```

Comment:
It is better to keep this `final` - any particular reason to change this ?

Reply:
https://github.com/apache/spark/pull/37624#discussion_r956560789